.github/workflows/release.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   Release:
     name: Release
-    runs-on: ubuntu-18.04
```

Comment:
According to https://github.com/model-checking/kani/pull/1149, this will make our binaries not work on Ubuntu 18.04. So, we should at least update the documentation to mention 18.04 is no longer supported.

Reply:
Will fix in followup PR https://github.com/model-checking/kani/issues/2197